translations/wesnoth-Towards_the_Light/cs.po

```diff
@@ -4,24 +4,24 @@ msgstr ""
 "Report-Msgid-Bugs-To: https://bugs.wesnoth.org/\n"
 "POT-Creation-Date: 2018-07-20 10:45 UTC\n"
 "PO-Revision-Date: \n"
-"Last-Translator: Jakub Konečný <jakub.konecny2@centrum.cz>\n"
-"Language-Team: none\n"
+"Last-Translator: Septim <septim@centrum.cz>\n"
+"Language-Team: Czech https://wiki.wesnoth.org/CzechTranslation\n"
 "Language: cs_CZ\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 2.0.9\n"
+"X-Generator: Poedit 2.4.2\n"
 "Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
 
 #. [heals]: id=healing {VALUE}
 #: Towards_the_Light\macros\abilities.cfg:8
 msgid "heals +"
-msgstr "léčení +"
+msgstr "léčí +"
```

Comment:
Tohle je konvence v oficiálním překladu? Myslím u jiných hodnot než 4 a 8.

Reply:
Ano. Teda na 100 % to ještě rozhodnuto není, ale nejspíš se půjde touto cestou i v případě hodnot 4 a 8.